assets/models/system/facebook-dinov2-image-embeddings-base/description.md

```diff
@@ -0,0 +1,50 @@
+The Vision Transformer (ViT) is a transformer encoder model (BERT-like) pretrained on a large collection of images in a self-supervised fashion.
+
+Images are presented to the model as a sequence of fixed-size patches, which are linearly embedded. One also adds a [CLS] token to the beginning of a sequence to use it for classification tasks. One also adds absolute position embeddings before feeding the sequence to the layers of the Transformer encoder.
+
+Note that this model does not include any fine-tuned heads.
+
+By pre-training the model, it learns an inner representation of images that can then be used to extract features useful for downstream tasks: if you have a dataset of labeled images for instance, you can train a standard classifier by placing a linear layer on top of the pre-trained encoder. One typically places a linear layer on top of the [CLS] token, as the last hidden state of this token can be seen as a representation of an entire image.
+
+### Intended uses & limitations
```

Comment:
FYI, we need to follow this specification for writing model description https://microsoftapc-my.sharepoint.com/:w:/r/personal/ayushmishra_microsoft_com/_layouts/15/Doc.aspx?sourcedoc=%7B2E13DB38-A1A9-4148-B1DC-AC14A76E1F72%7D&file=System%20Registry%20model%20card%20alignment.docx&action=default&mobileredirect=true&DefaultItemOpen=1&share=IQE42xMuqaFIQbHcrBSnbh9yAamNOJxrsZ9DvamUluUGp40

@novaturient95 please correct me if anything changed?

Reply:
Done.